src/asmcnc/production/z_head_qc_jig/z_head_qc_1.py

```diff
@@ -609,13 +609,13 @@ def temp_power_check(self, dt):
             fail_report.append("PCB Temperature: " + str(self.m.s.pcb_temp) + " degrees C")
             fail_report.append("Should be greater than 10 and less than 70 deg C.")
 
-        if 10 < self.m.s.motor_driver_temp < 60:
+        if 35 < self.m.s.motor_driver_temp < 100:
```

Comment:
Maybe store this as variables somewhere? Not important but helps readability & maintenance